bin/stripe-mock-server

```diff
@@ -11,6 +11,7 @@ opts = Trollop::options do
   opt :server, "Server to use", :type => :string, :default => 'thin'
   opt :debug, "Request and response output", :default => true
   opt :pid_path, "Location to put server pid file", :type => :string, :default => './stripe-mock-server.pid'
+  opt :require, "Extra path to require when loading the server; can be specified multiple times", :type => :string, :multi => true
```

Comment:
This, and the implementation in `lib/stripe_mock/server.rb` are part of #903 and can be ignored so far as this PR is concerned.